code/src/cljs/sixsq/nuvla/ui/credentials/events.cljs

```diff
@@ -95,6 +95,13 @@
                                      #{"credential-template/create-credential-vpn-customer"}
                                      (get-in new-credential [:template :href]))
                                    (dispatch [::set-generated-credential-modal %]))
+                                 (let [{:keys [status message resource-id]} (response/parse %)]
+                                   (dispatch [::get-results])
```

Comment:
This get-results event is not defined

Reply:
I don't remember touching that code...

anyway, I've removed the line and re-tested the SSH workflow and it works fine